packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -259,6 +259,8 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
       iconTitle: suggestion.isRecentQuery
         ? this.labels.recentQueryAriaLabel
         : this.labels.querySuggestionAriaLabel,
+      isClearRecentQueryButton: false,
+      isRecentQuery: !!suggestion.isRecentQuery,
```

Comment:
This is something I added that changes the behavior of the suggestion list event data.

If the selected suggestion was not the clearRecentQueryButton and/or not a recent query, we used to emit the event with these two properties being `undefined`. 

I now make them explicitely be `false` instead of `undefined`.

I don't think this is a breaking change but please think about it if you can see a case where this could be problematic.

Reply:
I found it weird that we were sometimes testing for these properties upstream, and they were either `true` or `undefined`.
It felt like it was something that had been forgotten when we added the recentQueries, to account for when the event is about a querySuggest instead.

I would push for clearer contracts between components instead of relying on `undefined` testing because there are properties that were omited. I think it's just a better practice.